rawkv/rawkv.go

```diff
@@ -67,8 +67,7 @@ const (
 )
 
 type scanOptions struct {
-	KeyOnly      bool
-	ColumnFamily string
+	KeyOnly bool
```

Comment:
I think the option is still useful. There could be some case that need to read/write different CFs. Creating multiple clients could waste more resource.